js/package.json

```diff
@@ -2,13 +2,13 @@
   "name": "surface",
   "version": "1.0.0",
   "description": "WIT IEEE Underwater ROV 2018 surface station",
-  "main": "ROV.js",
+  "main": "index.js",
   "scripts": {
     "test": "mocha test",
```

Comment:
That's a very good question. My gut says to put them in `js/surface` but since all the vision stuff is gonna be in Python it's gonna have to run in a different process than the surface station.

Lucky for us the actual dashboard talks to the surface station process using a websocket and `socket.io` which Python supports. So any text data can be sent straight from the vision scripts to the dashboard.

I'm thinking we also have the vision scripts host their _own_ video stream which the dashboard then reads from so we can see whatever extra shit the vision processing draws on top of the video.

Reply:
How about me? If the magnetometer, etc, code is done in JS then it could go in botserver.js- would that be the sensible place for it?

Reply:
yes